frontends/web/src/routes/new-settings/components/settingsItem/settingsItem.module.css

```diff
@@ -62,10 +62,6 @@
          min-width: 100%;
     }
 
-    .rightContentContainer {
-        margin-top: var(--space-quarter);
-    }
-
```

Comment:
hm removing this fixes the screenshot (on mobile) that I pasted, but then it doesn't have enough spacing in another place with full width dropdowns. Will update...

Reply:
Sure. Seems to be okay for now. If this breaks anything it'd be a simple fix anyway.